### ISSUE TITLE
Why is the reconstruction PSNR of dynamic scenes so much higher than that of static scenes? 

### ISSUE THREAD

**Issue by yifanlu0227** [fudan-zvg/PVG#2]
Thank you for your work, I am so impressed with your multi-view reconstruction result! 

I wonder why, with the same 5 cameras, the dynamic scene's PSNR of Table 1 is so much higher than the static scene of Table 2. Static scenes should be an easier case but show poor PSNR. Are there any differences in settings between the experiments on the two tables? 

I also conducted related experiments on Waymo, but when going from 3 cameras to 5 cameras, the reconstruction deteriorated significantly. Could you tell me which model design allows you to maintain the best results with 5 cameras?

**Comment by sonnefred** [comment]
> Thank you for your work, I am so impressed with your multi-view reconstruction result!
> 
> I wonder why, with the same 5 cameras, the dynamic scene's PSNR of Table 1 is so much higher than the static scene of Table 2. Static scenes should be an easier case but show poor PSNR. Are there any differences in settings between the experiments on the two tables?
> 
> I also conducted related experiments on Waymo, but when going from 3 cameras to 5 cameras, the reconstruction deteriorated significantly. Could you tell me which model design allows you to maintain the best results with 5 cameras?

Hi, have you tried to implement PVG by yourself? I'm also trying to implement it by myself, but the result is so poor and I'm not sure what caused this. Could you tell me how you implemented it? Thanks.

**Comment by yifanlu0227** [comment]
No, I applied vanilla 3DGS on the Waymo data. Since I just selected static scenes, those dynamic-object designs are not necessary. @sonnefred

**Comment by sonnefred** [comment]
> No, I applied vanilla 3DGS on the Waymo data. Since I just selected static scenes, those dynamic-object designs are not necessary. @sonnefred

Ok, thanks.

**Comment by Fumore** [comment]
@yifanlu0227 We are very sorry for the late reply. In the initial implementation, we normalized the camera poses. This leads to the selection threshold of clone and split being too large in the long drive scenario, resulting in a somewhat reduced reconstruction ability. Dynamic scenes we selected contains 50 frames while static scenes contains 100-200 frames with longer trajectory of the ego car. We fixed the shrinkage ratio of camera translation to solve this problem (the parameter --fix_radius). We will update our paper in arxiv, and the results of the dynamic and static scenes are similar.
